http4k-client/websocket/src/main/kotlin/org/http4k/client/internal/internal.kt

```diff
@@ -57,6 +57,8 @@ class NonBlockingClient(uri: Uri, headers: Headers, timeout: Duration, private v
 
     override fun onMessage(message: String) = socket.get().triggerMessage(WsMessage(message))
 
+    override fun onMessage(bytes: ByteBuffer) = socket.get().triggerMessage(WsMessage(Body(bytes.array().inputStream())))
```

Comment:
I don't like assuming what's in a `ByteBuffer`, but the `BlockingQueueClient` does the same, and will ensure the body is ` StreamBody` as expected.